code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java

```diff
@@ -0,0 +1,67 @@
+package code401Challenges.GetEdge;
+
+import code401Challenges.Graph.Edge;
+import code401Challenges.Graph.Graph;
+import code401Challenges.Graph.Node;
+
+import java.util.HashSet;
+
+public class GetEdge<T> {
+
+    boolean output = false;
+    int cost = 0;
+
+    public GetEdge(){}
+
+    public GetEdge(boolean output, int cost){
+        this.output = output;
+        this.cost = cost;
+    }
+
+    public GetEdge directFlights(Graph graph, String[] arr){
+// check if current is null, if so, return false
+
+        HashSet<Node> h = graph.getAdjNodesList();
+        Node current = null;
+        int totalCost = 0;
+
+        for(Node node : h){
+            if(node.label.equals(arr[0])){
+                current = node;
+            }
+        }
+
+
+        if(current == null){
+            GetEdge answer = new GetEdge(false, 0);
```

Comment:
Since you have `new GetEdge(false, 0)` in your code twice, why not save it to a variable near the top of the function and return it (changing it if it should be `true`)?